frontend/packages/kubevirt-plugin/integration-tests/tests/vm.migration.scenario.ts

```diff
@@ -60,19 +62,6 @@ describe('Test VM Migration', () => {
     VM_BOOTUP_TIMEOUT_SECS,
   );
 
-  it(
```

Comment:
why is this test removed?

Reply:
Decided with @jelkosz to remove this one as it is tested by migrating a VM twice in the following test case

Reply:
ok